### ISSUE TITLE
make all route string consts (and centralize them)

### ISSUE THREAD

**Issue by juanigaray** [edipoarg/radar#68]
See the Main2 and App components for reference.

**Comment by xaiki** [comment]
we name routes directly in-code, as in:
`<Link to="/some/route" />` this is error prone as typos might occur, and changing a route requires catching all different iterations.

this bug proposes migrating all routes to a central file and using this syntax instead:
```js
import {SomeRoute} from './routes';
<Link to={SomeRoute} />
```

**Comment by xaiki** [comment]
closed by #103
